addons/event_sale/wizard/event_configurator_views.xml

```diff
@@ -5,7 +5,15 @@
         <field name="model">event.event.configurator</field>
         <field name="arch" type="xml">
             <form js_class="event_configurator_form">
-                <group>
+                <field name="available_event_ticket_count" invisible="1"/>
+                <div invisible="available_event_ticket_count > 0">
+                    Please note that at least one ticket type must be set for an event. <br/>
+                    <a role="button" class="btn btn-link"
+                        href="/web#view_type=kanban&amp;model=event.event&amp;action=event.action_event_view">
+                        <i class="fa fa-arrow-right"/> Configure Events &amp; Tickets
+                    </a>
```

Comment:
Can't we use a regular action instead of a manually built link? Maybe an object action on the configurator model that re-uses an existing event action in its code, specifying to open the target in "new" (see previous comment).